downloads/github-actions-cheat-sheet.md

```diff
@@ -0,0 +1,193 @@
+
+- `ubuntu-latest`, `ubuntu-18.04`, or `ubuntu-16.04`
+- `windows-latest`, `windows-2019`, or `windows-2016`
+- `macOS-latest` or `macOS-10.14`
+
+#### `container`
+Instead of running directly on a host selected with `runs-on`, a container can run any steps in a job that don't already specify a container. If you have steps that use both script and container actions, the container actions will run as sibling containers on the same network with the same volume mounts. This object has the following attributes: `image`, `env`, `ports`, `volume` and `options`.
+
+```yaml
+...
+jobs:
+  my_job:
+    container:
+      image: node:10.16-jessie
+      env:
+        NODE_ENV: development
+      ports:
+        - 80
+      volumes:
+        - my_docker_volume:/volume_mount
+      options: --cpus 1
+```
+
+#### `services`
+Additional containers to host services for a job in a workflow. These are useful for creating databases or cache services. The runner on the virtual machine will automatically create a network and manage the lifecycle of the service containers. Each service is a named object in the `services` collection (`redis` or `nginx` for example) and can receive the same parameters than the `container` object.
+
+#### `timeout-minutes`
+The maximum number of minutes to let a workflow run before GitHub automatically cancels it. Default: 360
+
+{% endcapture %}
+<div class="col-md-6">
+{{ colTwo | markdownify }}
+</div>
+<div class="clearfix"></div>
+
+
+{% capture colThree %}
+### Job `steps`
+A job contains a sequence of tasks called `steps`. Steps can run commands, run setup tasks, or run an action from your repository, a public repository, or an action published in a Docker registry. Each step runs in its own process in the virtual environment and has access to the workspace and filesystem.
+
+#### Step `name`
+Specify the label to be displayed for this step in GitHub. It's not required but does improve readability in the logs.
+
+#### `uses`
+Select an action to run as part of a step in your job. You can use an action defined in the same repository as the workflow, a public repository elsewhere on GitHub, or in a published Docker container image.
+
+#### `with`
+A map of the input parameters defined by the action. Each input parameter is a key/value pair. Input parameters are prefixed with `INPUT_`, converted to upper case and set as environment variables: the `first_name` parameter will be exposed as the `INPUT_FIRST_NAME` environment variable. Special parameter names are:
+- `args`, a string that defines the inputs passed to a Docker container's `ENTRYPOINT`. It is used in place of the `CMD` instruction in a `Dockerfile`.
+- `entrypoint`, a string that defines or overrides the executable to run as the Docker container's `ENTRYPOINT`.
+
+#### `env`
+Sets environment variables for steps to use in the virtual environment.  If you are setting a secret in an environment variable, you must use the `secrets` context.
```

Comment:
A word on whether this is a safe practice would be helpful here.

Reply:
Not sure what to say here and whether it fits on a cheat sheet